templates/post-add.html

```diff
@@ -0,0 +1,59 @@
+{% extends 'base.html' %}
+{% load static %}
+
+{% block content %}
+<div class="container mt-5">
+    <h1 class="mb-4">Dodaj nowy post</h1>
+
+    {% if form.errors %}
+    <div class="alert alert-danger" role="alert">
+        <ul class="mb-0">
+            {% for field in form %}
+            {% for error in field.errors %}
+            <li>{{ error }}</li>
```

Comment:
Bit of indentation please, will be more readable.

Reply:
Actually, this seems like a duplicate, no? Later on you display errors next to field.

Reply:
done